contrib/cf-remote/cf_remote/packages.py

```diff
@@ -165,7 +165,11 @@ def __init__(self, edition="enterprise"):
             rel = Release(release)
             if "status" in release and release["status"] == "unsupported":
```

Comment:
So *master* doesn't have `status`? Isn't that just a matter of time before it does and things break again? I'd just do `if release["version"] != "master":...`.

Reply:
there is no "status" in https://cfengine.com/release-data/enterprise/releases.json, not anymore.

A brief of history: it was there, indeed, initially - when we thought that it would be great if `release.py` would specify which releases are supported, and which are not. However, back then `release.py` was running only during releases - and soon we realised that some versions can expire in-between releases. Hence, it was decided that `releases.json` file will include expiration dates for each branch, and it will be responsibility of the file reader to compare expiration dates with current dates to figure out which branches are supported. But "status" entries were left in the file for historical reasons, where @olehermanse saw them.